js/build_map.js

```diff
@@ -134,32 +163,70 @@ function fillMap() {
   updateCounties('USA');
 }
 
-function addButtons() {
-  var buttonContainer = d3.select('.svg-container')
-    .append('div')
-    .attr('id', 'button-container');
-    
-  buttonContainer.append('div')
-    .classed('select-arrowbox', true);
+function resize() {
+  
+  // Decide whether we're in mobile or desktop mode. Currently doing this by window width, but we could look to
+  // https://www.w3schools.com/howto/howto_js_media_queries.asp for more device-specific solutions
+  if(window.innerWidth > 425) { // sufficiently wide desktop windows
+    waterUseViz.mode = 'desktop';
```

Comment:
does `mode` need to exist in the `waterUseViz` global variable to start? It currently does not.

Reply:
It does not need to exist initially - this line will add it - but I like the idea of outlining the full global object all in one place, so I'll initialize it up top as `null`.